core/python/modconstruct.py

```diff
@@ -1,4 +1,4 @@
-from spt3g.core import G3Module, G3Pipeline, G3PipelineInfo, G3Frame, G3FrameType, G3Time, G3ModuleConfig, log_fatal
+from spt3g.core import G3Module, G3Pipeline, G3PipelineInfo, G3Frame, G3FrameType, G3Time, G3ModuleConfig, log_fatal, log_error
 try:
     from spt3g.core import multiprocess
     multiproc_avail = True
```

Comment:
Why not make multiproc_avail be false for Python > 3.7?

Reply:
changed this to raise an `ImportError` in `core/python/multiprocessing.py` instead